### PR TITLE
Blank Page for vue

### PR DESCRIPTION
**Pull request by SahilTara**
Summary of changes:
Implements the blank page for vue. You should now be able to generate a project with a blank page!

Should be reviewed after #647 

Just hit f5 and launch webts then hit generate! Dlls are in for windows and mac!
Related issues:
#199

### CODE REVIEW

**Review by Tanya0609 — COMMENTED** [review]
Try to use the change to generate a blank vue page, it works. Code looks good as well.

Will approve this PR once #631 is merged and this gets rebased

File: templates/Web/_composition/Vue/Page.Vue.AddConstantsFile/.template.config/template.json
Comment (jcoc611-microsoft) [inline templates/Web/_composition/Vue/Page.Vue.AddConstantsFile/.template.config/template.json]: Should indentation be fixed here?

File: templates/Web/_composition/Vue/Page.Vue.AddHome/.template.config/template.json
Comment (jcoc611-microsoft) [inline templates/Web/_composition/Vue/Page.Vue.AddHome/.template.config/template.json]: Same question about indentation

File: templates/Web/_composition/Vue/Page.Vue.AddConstantsFile/.template.config/template.json
Comment (Tanya0609) [reply templates/Web/_composition/Vue/Page.Vue.AddConstantsFile/.template.config/template.json]: I think the formatting issue is propagate from the same file in React. Also the other PRs are all interdependent on each other. I am concerned if change here will create merge conflicts for the remains of the PRs. Maybe we can have a separate PR to fix those formatting use Prettier together? Or you guys think it will be good to fix this one first?

Comment (SahilTara) [reply templates/Web/_composition/Vue/Page.Vue.AddConstantsFile/.template.config/template.json]: Yup this can be fixed. Actually all pages PRs are independent (Once vue base compositions is in my other PRs are independent).

File: templates/Web/_composition/Vue/Page.Vue.AddHome/.template.config/template.json
Comment (SahilTara) [reply templates/Web/_composition/Vue/Page.Vue.AddHome/.template.config/template.json]: Yup you're right I should fix this.

Comment (SahilTara) [reply templates/Web/_composition/Vue/Page.Vue.AddHome/.template.config/template.json]: Actually I noticed this is an issue for almost all of my template files in the repository. Do you think we should make a PR fixing formatting of all of these. I usually turn format on save off for templating because it can actually break certain templates (removes macros or edits certain lines). But we can make a giant PR fixing all template.jsons if you wish (which I think would be the better option (since this does not affect users)). Your call here though because I can fix it for all pages PRs right now.

File: templates/Web/_composition/Vue/Page.Vue.AddConstantsFile/.template.config/template.json
Comment (jcoc611-microsoft) [reply templates/Web/_composition/Vue/Page.Vue.AddConstantsFile/.template.config/template.json]: In that case yeah we can address the indent issue independently, thanks!

**Review by jasmingoh — APPROVED** [review]
Was able to generate blank pages and the code looks good!

**Pull request closed without merging**
